src/components/AddBook.jsx

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { v4 as uuidv4 } from 'uuid';
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+
+function AddBook({ handleAdd }) {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
[OPTIONAL] Instead of disabling the linter, consider resolving the issues it identifies. This helps you become more familiar with the tech's style guide.